pages/common/pueue-shutdown.md

```diff
@@ -0,0 +1,12 @@
+# pueue shutdown
+
+> Remotely shut down the daemon.
+> Only use this subcommand if the daemon isn't started by a service manager.
+
+- Shutdown the daemon without a service manager:
+
+`pueue shutdown`
+
+- Shutdown the daemon with systemd:
```

Comment:
I'm not sure if we should even include this example as it's not really demonstrating `pueue` functionality.  But if we do, it's good to make users aware that this particular command can only be run on Linux.
```suggestion
- Shutdown the daemon with systemd (Linux only):
```